config/module.go

```diff
@@ -213,3 +214,80 @@ func (m Module) EvaluateExePath(packagesDir string) (string, error) {
 	}
 	return m.ExePath, nil
 }
+
+// FirstRunSuccessExt is the extension of the file whose existence
+// denotes that the setup phase for a module ran successfully.
+const FirstRunSuccessExt = ".first_run_succeeded"
```

Comment:
instead of using the existing `.status.json` file, I opted to create a new file whose existence marks a successful setup phase run. this was driven by two factors:

* having a separate files avoids any issues of potentially corrupting the `.status.json` file
* importing `packages` in this file (to get the defined `.status.json` extension) creates a circular import

Reply:
> immortalize 

sounds good - will doc this now that there’s some consensus 

> extension 

it is an extension - the full file ends up being the versioned module name + this extension

Reply:
i'm gonna rename the variable to `Suffix` instead - i guess it's a bit weird to call `.first_run_succeeded` an "extension"